README.md

```diff
@@ -1,4 +1,4 @@
-# Nabaztag en Python pour Raspberry Pi
+# [Nabaztag en Python pour Raspberry Pi](https://github.com/nabaztag2018/pynab/releases)
```

Comment:
If you add a link, text should reflect on link. Here it's not the case.

`[pynab releases](https://github.com/nabaztag2018/pynab/releases)`

Reply:
> If you add a link, text **should** reflect of link.

Why?
In any case, we should not change the document title  from Nabaztag en _Python pour Raspberry Pi_ to _pynab releases_.
But the link may be removed, if the powers in charge prefer it so.